README.md

```diff
@@ -202,6 +202,49 @@ If you want to commit generated artifacts in the release commit (e.g. [#96](http
 "release": "git add <file(s) to commit> && standard-version -a"
 ```
 
+### Using a specific host context
+
+Running `standard-version` with the flag `--context` or `-c` allows you to specify a different host context.
+
+```sh
+# npm run script
+npm run release -- --context <context.json>
+# or global bin
+standard-version --context <context.json>
+```
+
+In case you are using a private version control system installed locally like gitlab, and your repository url does not contain the git type keyword (github|bitbucket|gitlab). It is mandatory to inform manually `standard-version` on the host context (keywords) to be used.
+
+`package.json` :
+```json
+{
+  "name": "Private Project",
+  "author": "Private Owner",
+  "version": "1.0.0",
+  "repository": "http:://private-repo.company.org/private-owner/private-project.git"
+}
+```
+`gitlab.json` :
```

Comment:
Would it be possible to provide a link to something that describes what options are available/expected in a context file?

Perhaps use this: https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-writer#context

(Seems like that link might not even accurately describe all options?)